biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblParser.java

```diff
@@ -0,0 +1,215 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * This class should parse the data of embl file
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblParser {
+
+    private File file;
+    private EmblId emblId = new EmblId();
+    private EmblReference emblReference = new EmblReference();
+    private List<String> accessionNumber = new LinkedList<>();
+    private String ProjectIdentifier;
+    private String OrGanelle;
+    private String createdDate;
+    private String featureHeader;
+    private String featureTable;
+    private String lastUpdatedDate;
+    private String sequenceDescription;
+    private List<String> keyword = new LinkedList<>();
+    private String organismSpecies;
+    private String organismClassification;
+    private String databaseCrossReference;
+    private String assemblyHeader;
+    private String assemblyInformation;
+    private String CON;
+    private String sequenceHeader;
+    private StringBuilder sequence = new StringBuilder("");
+
+
+    public EmblId getEmblId() {
+        return emblId;
+    }
+
+    public EmblReference getEmblReference() {
+        return emblReference;
+    }
+
+    public List<String> getAccessionNumber() {
+        return accessionNumber;
+    }
+
+    public String getProjectIdentifier() {
+        return ProjectIdentifier;
+    }
+
+    public String getCreatedDate() {
+        return createdDate;
+    }
+
+    public String getLastUpdatedDate() {
+        return lastUpdatedDate;
+    }
+
+    public String getSequenceDescription() {
+        return sequenceDescription;
+    }
+
+    public List<String> getKeyword() {
+        return keyword;
+    }
+
+    public String getOrganismSpecies() {
+        return organismSpecies;
+    }
+
+    public String getOrganismClassification() {
+        return organismClassification;
+    }
+
+
+    public String getDatabaseCrossReference() {
+        return databaseCrossReference;
+    }
+
+    public String getAssemblyHeader() {
+        return assemblyHeader;
+    }
+
+
+    public String getAssemblyInformation() {
+        return assemblyInformation;
+    }
+
+
+    public String getCON() {
+        return CON;
+    }
+
+
+    public String getSequenceHeader() {
+        return sequenceHeader;
+    }
+
+
+    public StringBuilder getSequence() {
+        return sequence;
+    }
+
+
+    public EmblParser(File file) {
+        setFile(file);
+    }
+
+    public void setFile(File file) {
+        if (file == null)
+            throw new NullPointerException("file can't be null");
+        this.file = file;
+    }
+
+    public void parse() {
```

Comment:
This method should throw an IOException instead of catching it. Otherwise the user would not be able to handle it properly in their own application.